client/src/components/Header/index.tsx

```diff
@@ -87,6 +88,7 @@ export class Header extends React.Component<
           <style>{':root{--header-height: 38px}'}</style>
         </Helmet>
         <header>
+          <SkipToContentButton />
```

Comment:
there is no need to DRY the button, it should be used only once and if it's used multiple times in the future, then we need to change the structure of the page instead of DRY the button.

```suggestion
<a href='#content-start' className='skip-to-content-button'>Skip To Content</a>
```